trino/sqlalchemy/dialect.py

```diff
@@ -106,6 +111,10 @@ def create_connect_args(self, url: URL) -> Tuple[Sequence[Any], Mapping[str, Any
             kwargs["http_scheme"] = "https"
             kwargs["auth"] = CertificateAuthentication(unquote_plus(url.query['cert']), unquote_plus(url.query['key']))
 
+        if "externalAuthentication" in url.query:
+            kwargs["http_scheme"] = "https"
```

Comment:
why automatically set this? Someone can have configured server to accept credentials over http (it's not recommended but it'll break someone).

Reply:
ingore